### ISSUE TITLE
TorrentDay not working

### ISSUE THREAD

**Issue by jappish84** [Jackett/Jackett#8876]
### Environment

**OS**: Diepti Virtualbox on Win 10

**.Net Runtime**: not sure

**.Net Version**: not sure

**Jackett Version**: 0.16.609.0

**Last Working Jackett Version**: some old =?

**Are you using a proxy or VPN?** no

### Description

Can't add torrentday as indexer, I found a couple of not so old closed issues that were referring that tday.love needing to be updated to torrentday.cool and that this would resolve the issue, but the issue persist. Could this issue be related to TD using cloudflare?

### Logged Error Messages

`curl -vvv https://tday.love/`

```
* Expire in 0 ms for 6 (transfer 0x56053aa2fe80)
* Expire in 1 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 0 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 2 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 0 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 0 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 2 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 0 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 0 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 2 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 0 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 0 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 2 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 1 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 1 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 4 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 1 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 1 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 4 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 1 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 2 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 4 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 2 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 2 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 4 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 3 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 3 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 4 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 3 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 3 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 4 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 4 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 4 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 4 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 4 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 4 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 4 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 5 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 5 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 4 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 6 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 6 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 8 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 8 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 8 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 8 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 9 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 9 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 16 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 12 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 12 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 16 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 12 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 12 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 16 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 16 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 16 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 16 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 16 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 16 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 16 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 50 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 50 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 16 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 50 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 50 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 32 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 50 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 50 ms for 1 (transfer 0x56053aa2fe80)
* Expire in 50 ms for 1 (transfer 0x56053aa2fe80)
*   Trying 104.27.156.102...
* TCP_NODELAY set
* Expire in 149941 ms for 3 (transfer 0x56053aa2fe80)
* Expire in 200 ms for 4 (transfer 0x56053aa2fe80)
* Connected to tday.love (104.27.156.102) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*   CAfile: none
  CApath: /etc/ssl/certs
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
* TLSv1.3 (IN), TLS handshake, Server hello (2):
* TLSv1.3 (IN), TLS handshake, Encrypted Extensions (8):
* TLSv1.3 (IN), TLS handshake, Certificate (11):
* TLSv1.3 (IN), TLS handshake, CERT verify (15):
* TLSv1.3 (IN), TLS handshake, Finished (20):
* TLSv1.3 (OUT), TLS change cipher, Change cipher spec (1):
* TLSv1.3 (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / TLS_AES_256_GCM_SHA384
* ALPN, server accepted to use h2
* Server certificate:
*  subject: C=US; ST=CA; L=San Francisco; O=Cloudflare, Inc.; CN=sni.cloudflaressl.com
*  start date: Dec  8 00:00:00 2019 GMT
*  expire date: Oct  9 12:00:00 2020 GMT
*  subjectAltName: host "tday.love" matched cert's "tday.love"
*  issuer: C=US; ST=CA; L=San Francisco; O=CloudFlare, Inc.; CN=CloudFlare Inc ECC CA-2
*  SSL certificate verify ok.
* Using HTTP2, server supports multi-use
* Connection state changed (HTTP/2 confirmed)
* Copying HTTP/2 data in stream buffer to connection buffer after upgrade: len=0
* Using Stream ID: 1 (easy handle 0x56053aa2fe80)
> GET / HTTP/2
> Host: tday.love
> User-Agent: curl/7.64.0
> Accept: */*
> 
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* old SSL session ID is stale, removing
* Connection state changed (MAX_CONCURRENT_STREAMS == 256)!
< HTTP/2 302 
< date: Fri, 05 Jun 2020 20:47:20 GMT
< content-length: 0
< set-cookie: __cfduid=d2b88ba605ddd37855e0811d83c60c34f1591390040; expires=Sun, 05-Jul-20 20:47:20 GMT; path=/; domain=.tday.love; HttpOnly; SameSite=Lax; Secure
< location: /login.php
< cf-cache-status: DYNAMIC
< cf-request-id: 0327d60a3f00007363e6848200000001
< expect-ct: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
< server: cloudflare
< cf-ray: 59ecbf89fead7363-CPH
< 
```

`openssl s_client -connect tday.love:443`

```
CONNECTED(00000003)
depth=2 C = IE, O = Baltimore, OU = CyberTrust, CN = Baltimore CyberTrust Root
verify return:1
depth=1 C = US, ST = CA, L = San Francisco, O = "CloudFlare, Inc.", CN = CloudFlare Inc ECC CA-2
verify return:1
depth=0 C = US, ST = CA, L = San Francisco, O = "Cloudflare, Inc.", CN = sni.cloudflaressl.com
verify return:1
---
Certificate chain
 0 s:C = US, ST = CA, L = San Francisco, O = "Cloudflare, Inc.", CN = sni.cloudflaressl.com
   i:C = US, ST = CA, L = San Francisco, O = "CloudFlare, Inc.", CN = CloudFlare Inc ECC CA-2
 1 s:C = US, ST = CA, L = San Francisco, O = "CloudFlare, Inc.", CN = CloudFlare Inc ECC CA-2
   i:C = IE, O = Baltimore, OU = CyberTrust, CN = Baltimore CyberTrust Root
---
Server certificate
-----BEGIN CERTIFICATE-----
MIIE4TCCBIagAwIBAgIQAgEE7s6mYV+bN/ZfWwVj9jAKBggqhkjOPQQDAjBvMQsw
CQYDVQQGEwJVUzELMAkGA1UECBMCQ0ExFjAUBgNVBAcTDVNhbiBGcmFuY2lzY28x
GTAXBgNVBAoTEENsb3VkRmxhcmUsIEluYy4xIDAeBgNVBAMTF0Nsb3VkRmxhcmUg
SW5jIEVDQyBDQS0yMB4XDTE5MTIwODAwMDAwMFoXDTIwMTAwOTEyMDAwMFowbTEL
MAkGA1UEBhMCVVMxCzAJBgNVBAgTAkNBMRYwFAYDVQQHEw1TYW4gRnJhbmNpc2Nv
MRkwFwYDVQQKExBDbG91ZGZsYXJlLCBJbmMuMR4wHAYDVQQDExVzbmkuY2xvdWRm
bGFyZXNzbC5jb20wWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAAQGHa6um7ZJwSUP
hYxYIBvbu8xuzeMpVXaagJsqOQ83E2fdK8ypS0MeOolAEhj/wvwmrt//SYI9gq03
emKnWDIKo4IDBDCCAwAwHwYDVR0jBBgwFoAUPnQtH89FdQR+P8Cihz5MQ4NRE8Yw
HQYDVR0OBBYEFF+F7H/lXVnzBqcbeIJ97EA+U1NhMDgGA1UdEQQxMC+CFXNuaS5j
bG91ZGZsYXJlc3NsLmNvbYILKi50ZGF5LmxvdmWCCXRkYXkubG92ZTAOBgNVHQ8B
Af8EBAMCB4AwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMCMHkGA1UdHwRy
MHAwNqA0oDKGMGh0dHA6Ly9jcmwzLmRpZ2ljZXJ0LmNvbS9DbG91ZEZsYXJlSW5j
RUNDQ0EyLmNybDA2oDSgMoYwaHR0cDovL2NybDQuZGlnaWNlcnQuY29tL0Nsb3Vk
RmxhcmVJbmNFQ0NDQTIuY3JsMEwGA1UdIARFMEMwNwYJYIZIAYb9bAEBMCowKAYI
KwYBBQUHAgEWHGh0dHBzOi8vd3d3LmRpZ2ljZXJ0LmNvbS9DUFMwCAYGZ4EMAQIC
MHYGCCsGAQUFBwEBBGowaDAkBggrBgEFBQcwAYYYaHR0cDovL29jc3AuZGlnaWNl
cnQuY29tMEAGCCsGAQUFBzAChjRodHRwOi8vY2FjZXJ0cy5kaWdpY2VydC5jb20v
Q2xvdWRGbGFyZUluY0VDQ0NBLTIuY3J0MAwGA1UdEwEB/wQCMAAwggEEBgorBgEE
AdZ5AgQCBIH1BIHyAPAAdQCkuQmQtBhYFIe7E6LMZ3AKPDWYBPkb37jjd80OyA3c
EAAAAW7l0NCKAAAEAwBGMEQCIC4ifvofa2re4jFvQ+hI+v9wkpioRad4vXmD5tgd
WW1HAiAtYrcFNNEL6Kiw77RgVjRHd4muxJVfpmejTwkNngs6xQB3AF6nc/nfVsDn
tTZIfdBJ4DJ6kZoMhKESEoQYdZaBcUVYAAABbuXQ0EIAAAQDAEgwRgIhAKe6Od4r
Rju6bDpRA3A1gRkVRNmr1RpdHOwluyBM8g95AiEA2xEIjcBmFRANXA1rG8KnR/ne
KcFhOaw/X0H9RKunlTEwCgYIKoZIzj0EAwIDSQAwRgIhAPbZYshYTjxCtnwcSdTw
JKhhKUuiAfWqMqexhfkBlfPHAiEA/VzDYS/zq30W5+7HBgLgsDY0JFJ0tr0lYJt/
CcwQtRU=
-----END CERTIFICATE-----
subject=C = US, ST = CA, L = San Francisco, O = "Cloudflare, Inc.", CN = sni.cloudflaressl.com

issuer=C = US, ST = CA, L = San Francisco, O = "CloudFlare, Inc.", CN = CloudFlare Inc ECC CA-2

---
No client certificate CA names sent
Peer signing digest: SHA256
Peer signature type: ECDSA
Server Temp Key: X25519, 253 bits
---
SSL handshake has read 2503 bytes and written 381 bytes
Verification: OK
---
New, TLSv1.3, Cipher is TLS_AES_256_GCM_SHA384
Server public key is 256 bit
Secure Renegotiation IS NOT supported
Compression: NONE
Expansion: NONE
No ALPN negotiated
Early data was not sent
Verify return code: 0 (ok)
```

`journalctl -u jackett.service`

```
jun 05 21:42:44 DietpiVM jackett[31392]: 06-05 21:42:44 Error CloudflareSolverRe.Exceptions.CloudflareClearanceException: Clearance failed after 10 attempt(s).
jun 05 21:42:44 DietpiVM jackett[31392]:    at CloudflareSolverRe.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
jun 05 21:42:44 DietpiVM jackett[31392]:    at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Bool
ean disposeCts)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Jackett.Common.Utils.Clients.HttpWebClient2NetCore.Run(WebRequest webRequest) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Cli
ents/HttpWebClient2NetCore.cs:line 229
jun 05 21:42:44 DietpiVM jackett[31392]:    at Jackett.Common.Utils.Clients.WebClient.GetString(WebRequest request) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Clients/WebC
lient.cs:line 114
jun 05 21:42:44 DietpiVM jackett[31392]:    at Jackett.Common.Indexers.BaseWebIndexer.RequestStringWithCookies(String url, String cookieOverride, String referer, Dictionary`2 h
eaders) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 443
jun 05 21:42:44 DietpiVM jackett[31392]:    at Jackett.Common.Indexers.TorrentDay.GetConfigurationForSetup() in /home/vsts/work/1/s/src/Jackett.Common/Indexers/TorrentDay.cs:li
ne 126
jun 05 21:42:44 DietpiVM jackett[31392]:    at Jackett.Server.Controllers.IndexerApiController.Config() in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiControll
er.cs:line 79
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, 
ObjectMethodExecutor executor, Object controller, Object[] arguments)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker 
invoker, ValueTask`1 actionResultValueTask)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvo
ker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isComplet
ed)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker i
nvoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task 
lastTask, State next, Scope scope, Object state, Boolean isCompleted)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposa
ble scope)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logg
er)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middle
ware/CustomExceptionHandler.cs:line 26 CloudflareSolverRe.Exceptions.CloudflareClearanceException: Clearance failed after 10 attempt(s).
jun 05 21:42:44 DietpiVM jackett[31392]:    at CloudflareSolverRe.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
jun 05 21:42:44 DietpiVM jackett[31392]:    at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Bool
ean disposeCts)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Jackett.Common.Utils.Clients.HttpWebClient2NetCore.Run(WebRequest webRequest) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Cli
ents/HttpWebClient2NetCore.cs:line 229
jun 05 21:42:44 DietpiVM jackett[31392]:    at Jackett.Common.Utils.Clients.WebClient.GetString(WebRequest request) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Clients/WebC
lient.cs:line 114
jun 05 21:42:44 DietpiVM jackett[31392]:    at Jackett.Common.Indexers.BaseWebIndexer.RequestStringWithCookies(String url, String cookieOverride, String referer, Dictionary`2 h
eaders) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 443
jun 05 21:42:44 DietpiVM jackett[31392]:    at Jackett.Common.Indexers.TorrentDay.GetConfigurationForSetup() in /home/vsts/work/1/s/src/Jackett.Common/Indexers/TorrentDay.cs:li
ne 126
jun 05 21:42:44 DietpiVM jackett[31392]:    at Jackett.Server.Controllers.IndexerApiController.Config() in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiControll
er.cs:line 79
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, 
ObjectMethodExecutor executor, Object controller, Object[] arguments)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker 
invoker, ValueTask`1 actionResultValueTask)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvo
ker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isComplet
ed)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker i
nvoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task 
lastTask, State next, Scope scope, Object state, Boolean isCompleted)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposa
ble scope)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logg
er)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
jun 05 21:42:44 DietpiVM jackett[31392]:    at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middle
ware/CustomExceptionHandler.cs:line 26
```

**Comment by garfield69** [comment]
your jackett log snippet should that your access to torrentday was blocked by the cloudflare anti-ddos check that is plaguing other Jackett indexers.
closing as a duplicate of the open issue #8840 for which there is not current solve.

**Comment by jappish84** [comment]
Alright, thanks @garfield69 I misinterpreted the issue, sorry :/
